test/integration/content-explorer/ContentExplorer.e2e.test.js

```diff
@@ -0,0 +1,207 @@
+// <reference types="Cypress" />
+const helpers = {
+    load(additionalProps = { canUpload: true, permissions: { canUpload: true } }) {
```

Comment:
Is `permissions` exposed as a top-level prop in ContentExplorer?

Reply:
No, it's not.  That line is an artifact of me trying to get permissions to work for testing.  I'll take it out in the next commit.